presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetHiveRecordCursor.java

```diff
@@ -225,23 +224,7 @@ public long getTotalBytes()
     @Override
     public long getCompletedBytes()
```

Comment:
`getCompletedBytes()` returns the uncompressed size and `getTotalBytes()` returns the compressed size, and this contradicts with the previous implementation where `completedBytes < totalBytes`. I don't know whether this may have a side effect elsewhere (or in the future where some code depends on this assumption).